static/app/views/dashboards/manage/dashboardList.tsx

```diff
@@ -160,14 +178,30 @@ function DashboardList({
   }
 
   function renderDashboardGrid() {
-    if (!dashboards?.length) {
+    if (!currentDashboards?.length) {
       return (
         <EmptyStateWarning>
           <p>{t('Sorry, no Dashboards match your filters.')}</p>
         </EmptyStateWarning>
       );
     }
-    return <DashboardGrid>{renderMiniDashboards()}</DashboardGrid>;
+    const DashboardGrid = styled('div')`
+      display: grid;
+      grid-template-columns: repeat(${columns}, minmax(300px, 1fr));
+      grid-template-rows: repeat(${rows}, max-content);
+      gap: ${space(2)};
+    `;
```

Comment:
This is a bit suspicious, could you move this out of this component? It's usually not good to dynamically create components inside other components like this